PyInstaller/hooks/hook-_tkinter.py

```diff
@@ -77,7 +78,7 @@ def _warn_if_activetcl_or_teapot_installed(tcl_root, tcltree):
 
     mentions_activetcl = False
     mentions_teapot = False
-    with open(init_resource, 'r') as init_file:
+    with open_file(init_resource, text_read_mode, encoding='utf-8') as init_file:
```

Comment:
Good point. I don't know the encoding. The existing code assumes an encoding: ASCII for Python 2, and `locale.getpreferredencoding()` for Python 3. While this has (hopefully) worked in the past, it seems poor practice for the future. Do you have any thoughts? I think we should explicitly specify an encoding, rather than relying on this "default" which is Python- and platform-dependent.

Reply:
Well, explicit is better than implicit :-) . But when reading https://www.tcl.tk/doc/howto/i18n.html#system_encoding, I assume `locale.getpreferredencoding()` is the right thing to do. I suggest using this explicitly and refering to the mentioned url.

Reply:
Great! Done.